src/common/tse.c

```diff
@@ -448,6 +448,7 @@ tse_task_prep_callback(tse_task_t *task)
 	struct tse_task_private	*dtp = tse_task2priv(task);
 	struct tse_task_cb	*dtc;
 	struct tse_task_cb	*tmp;
+	bool			 ret = true;
 	int			 rc;
 
 	d_list_for_each_entry_safe(dtc, tmp, &dtp->dtp_prep_cb_list, dtc_list) {
```

Comment:
not related to the patch, but I have a personal preference for

while ((dtc = d_list_pop_entry(&dtp->dtp_prep_cb_list, dtc_list)) != NULL) {
    ...
}

since it doesn't require the extra pointer